.github/workflows/wheel_tests_and_release.yml

```diff
@@ -20,8 +20,9 @@ jobs:
     runs-on: ubuntu-latest
     environment: release
     permissions:
-      contents: write # for softprops/action-gh-release to create GitHub release
+      contents: write # for softprops/action-gh-release to create GitHub release and for GitHub attestations
```

Comment:
GitHub attestations only need `read` access, so having `write` is more than is needed for them, but still just noting they are related to this permission.

Reply:
That's quite confusing then; I'd remove this comment, since it has the permissions it needs by default. Otherwise, make explicit that attestations only need read permissions.

Reply:
Sounds good, and removed (pushed back in a squash with a `--force-with-lease` &mdash; sorry if that happens to not the preferred way for reviewing).